modules/ROOT/pages/passing-additional-arguments-to-the-jvm-to-control-mule.adoc

```diff
@@ -0,0 +1,21 @@
+= Passing Additional Arguments to the JVM to Control Mule
```

Comment:
I almost wonder if this should be a section in some other topic rather than its own topic. Not sure where though...